Sources/TuistCache/GraphMappers/CacheTreeShakingGraphMapper.swift

```diff
@@ -75,21 +75,26 @@ public final class CacheTreeShakingGraphMapper: GraphMapping {
 
     fileprivate func treeShake(schemes: [Scheme], sourceTargets: Set<TargetReference>) -> [Scheme] {
         schemes.compactMap { scheme -> Scheme? in
-            let buildActionTargets = scheme.buildAction?.targets.filter { sourceTargets.contains($0) } ?? []
+            var focusedScheme = scheme
```

Comment:
Nitpick: I think it'd be more accurate to keep this one as scheme. What the user focus on is a target and not a scheme.

```swift
var scheme = scheme
```

Reply:
Good point! Dropping `focused`